module/evdi_drv.c

```diff
@@ -114,11 +116,44 @@ static void evdi_add_device(void)
 	evdi_context.dev_count++;
 }
 
+#if KERNEL_VERSION(4, 12, 0) >= LINUX_VERSION_CODE
```

Comment:
For kernels older than **4.12.0** we want the *old* version, and from **4.12.0** *new*, right?
Now consider few cases:
- **4.10.0**
```KERNEL_VERSION(4, 12, 0) >= 4.10.0``` gives ```true```, the *new* version will be used. Incorrect.
- **4.12.0**
```KERNEL_VERSION(4, 12, 0) >= 4.12.0``` gives ```true```, the *new* version will be used. OK.
- **4.12.1**
```KERNEL_VERSION(4, 12, 0) >= 4.12.1``` gives ```false```, the *old* version will be used. Incorrect